tests/assets.spec.ts

```diff
@@ -109,10 +115,10 @@ describe('ASSETS', () => {
         await assets.tx.approveTransfer(ASSET_ID, bob.address, 100, {gasLimit: gas3 });
 
         // @ts-ignore
-        await expect((await assets.query.allowance(ASSET_ID, alice.address, bob.address)).value.toNumber()).to.equal(100)
+        await expect((await assets.query.allowance(ASSET_ID, alice.address, bob.address)).value.unwrap().toNumber()).to.equal(100)
```

Comment:
can you use` ?` instead of `unwrap()`? Just asking

Reply:
Typescript don't have rust like escape '?' operator.